Header header
float32 angle_min
float32 angle_max
float32 angle_inc
float32 time_inc
float32 range_min
float32 range_max
float32[] angles
float32[] ranges
